### PR TITLE
Adjust rnpm link to colorize skipped linking steps

### PR DESCRIPTION
**Pull request by GantMan**
Sometimes when working with a team of developers, someone adds a library but does not link it. To identify if this required linking, you have to type react-native link and then read a wall of text to see if rnpm handled the issue. This can be sped up with a friendly logging level for previously linked messages.

Continuation of #9507 

### Changes This:
image

### Into This:
image

### CODE REVIEW

**Comment by facebook-github-bot-3** [comment]
By analyzing the blame information on this pull request, we identified @grabbou to be a potential reviewer.

**Comment by facebook-github-bot-2** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/c32af7f105a72dfc3067faa0216a07c1e1c0edc9..3a821f2a00adec1d33407d33c437ff408c68e2a7)

**Comment by facebook-github-bot-1** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/3a821f2a00adec1d33407d33c437ff408c68e2a7..cff24f7974f62a83f8388c581abd626cadfc9a6f)

**Comment by facebook-github-bot-3** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/cff24f7974f62a83f8388c581abd626cadfc9a6f..dd225c72ea71cffaa82ec5888597d5ff0606a5cf)

**Comment by facebook-github-bot-9** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/dd225c72ea71cffaa82ec5888597d5ff0606a5cf..181826a799825173a15160a00b4fdb6684c5857c)

**Comment by GantMan** [comment]
I'm at a loss on how to pass these tests @grabbou

**Comment by GantMan** [comment]
cc: @grabbou  @satya164  - any direction on why this is failing tests would be awesome.   I'd love to take care of whatever it is.   Is there good documentation online on how I can run these tests locally?

**Comment by facebook-github-bot-3** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/181826a799825173a15160a00b4fdb6684c5857c..13bfa2faf81916a7b33ac0a11468be138bf66b15)

**Comment by facebook-github-bot-9** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/a170fc233310178655f12e7d42172dea3ef067f6..a80d04d47de5918206e5c61171b7dca1987f0e89)

**Comment by facebook-github-bot-9** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/a80d04d47de5918206e5c61171b7dca1987f0e89..f0af996c949081d342585f00752adbcf88d8e71f)

**Comment by GantMan** [comment]
sorry for the noise in this thread, I believe I understand the problem but have no easy way to test without pushing.

**Comment by facebook-github-bot-0** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/f0af996c949081d342585f00752adbcf88d8e71f..236f7e9b82b6044170c6d9224a5c1e776de02c21)

**Comment by GantMan** [comment]
@grabbou  I finally got down to it and fixed the tests.  The mock folder was turned off in these particular tests, so I had to set the mock manually.

@bestander It took a strange winding road but we got it.  The issue from #9507 has been resolved with tests.  Please merge when you guys are satisfied.

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3821468) to review internal test results.

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3821468) to review internal test results.

**Comment by facebook-github-bot-2** [comment]
I tried to merge this pull request into the Facebook internal repo but some checks failed. To unblock yourself please check the following: Does this pull request pass all open source tests on GitHub? If not please fix those. Does the code still apply cleanly on top of GitHub master? If not can please rebase. In all other cases this means some internal test failed, for example a part of a fb app won't work with this pull request. I've added the Import Failed label to this pull request so it is easy for someone at fb to find the pull request and check what failed. If you don't see anyone comment in a few days feel free to comment mentioning one of the core contributors to the project so they get a notification.

**Comment by GantMan** [comment]
Looks like I was able to merge it with master no problem.  Not sure what must have failed.

**Comment by facebook-github-bot-4** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/236f7e9b82b6044170c6d9224a5c1e776de02c21..fcf7ec613ba236010408b74426597cbc16e58407) - [changes since last import](https://github.com/facebook/react-native/pull/9551/files/236f7e9b82b6044170c6d9224a5c1e776de02c21..fcf7ec613ba236010408b74426597cbc16e58407)

**Comment by bestander** [comment]
Something went wrong, let's try again

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-1** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3821468) to review internal test results.

**Comment by facebook-github-bot-1** [comment]
I tried to merge this pull request into the Facebook internal repo but some checks failed. To unblock yourself please check the following: Does this pull request pass all open source tests on GitHub? If not please fix those. Does the code still apply cleanly on top of GitHub master? If not can please rebase. In all other cases this means some internal test failed, for example a part of a fb app won't work with this pull request. I've added the Import Failed label to this pull request so it is easy for someone at fb to find the pull request and check what failed. If you don't see anyone comment in a few days feel free to comment mentioning one of the core contributors to the project so they get a notification.

**Comment by facebook-github-bot-1** [comment]
@GantMan updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9551/files/fcf7ec613ba236010408b74426597cbc16e58407..099447ae088cf2713564f3a8da17aaac923ba64c) - [changes since last import](https://github.com/facebook/react-native/pull/9551/files/fcf7ec613ba236010408b74426597cbc16e58407..099447ae088cf2713564f3a8da17aaac923ba64c)

**Comment by bestander** [comment]
Trunk was broken yesterday, let's do once more.
Otherwise I'll call the cavalry

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://phabricator.intern.facebook.com/D3821468) to review internal test results.

File: local-cli/__mocks__/chalk.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (bestander) [inline local-cli/__mocks__/chalk.js]: I think I got the reason why it fails to merge.
We have another chalk.js mock: react-native-github/packager/react-packager/src/Activity/__mocks__/chalk.js.

@GantMan, could you merge them into one?

Comment (bestander) [inline local-cli/__mocks__/chalk.js]: This is the error message
```
 ● Test suite failed to run

    TypeError: Cannot read property 'bold' of undefined
      
      at Object.<anonymous> (react-native-github/node_modules/babel-code-frame/lib/index.js:69:31)
```

**Comment by GantMan** [comment]
Everything passes in CI, I'm at a loss on how to get this contribution in.   @grabbou - if you'd like, you can add this code internally and just close this PR.  I don't have to get any credit, I just don't know how else to contribute.

**Comment by bestander** [comment]
Sorry, @GantMan, this is our internal tests that got broken by the mock.
I'll try to have a look myself why it happens when I have a chance.
Thanks for the contribution anyway.

**Comment by mkonicek** [comment]
Hmm, is there any way to implement this without changing the `local-cli/__mocks__/chalk.js`?

**Comment by mkonicek** [comment]
I wanted to assign this to @bestander but then realized he reviewed already.

**Comment by facebook-github-bot-9** [comment]
I tried to merge this pull request into the Facebook internal repo but some checks failed. To unblock yourself please check the following: Does this pull request pass all open source tests on GitHub? If not please fix those. Does the code still apply cleanly on top of GitHub master? If not can please rebase. In all other cases this means some internal test failed, for example a part of a fb app won't work with this pull request. I've added the Import Failed label to this pull request so it is easy for someone at fb to find the pull request and check what failed. If you don't see anyone comment in a few days feel free to comment mentioning one of the core contributors to the project so they get a notification.

**Pull request closed without merging**